ui/src/app/pages/signed-in/inactivity-monitor.tsx

```diff
@@ -2,13 +2,33 @@ import * as React from 'react';
 
 import { environment } from 'environments/environment';
 import { withErrorModal } from 'app/components/modals';
-import { TextModal } from 'app/components/text-modal';
-import { debouncer } from 'app/utils';
+import { InactivityModal } from 'app/pages/signed-in/inactivity-modal';
 import { AnalyticsTracker } from 'app/utils/analytics';
 import { signOut } from 'app/utils/authentication';
+import { authStore, useStore } from 'app/utils/stores';
 
 const { useState, useEffect } = React;
 
+// Returns a function which will execute `action` at most once every `sensitivityMs` milliseconds
+// if the returned function has been invoked within the last `sensitivityMs` milliseconds
+export function debouncer(action, sensitivityMs) {
```

Comment:
we also use `useDebounce` and `fp.debounce` - this seems like a good opportunity for consolidation

Reply:
Good to know!